quickwit/quickwit-serve/src/tests.rs

```diff
@@ -118,6 +120,9 @@ async fn test_multi_nodes_cluster() {
 
     assert!(sandbox.rest_client.is_live().await.unwrap());
 
+    // Wait until indexing pipelines are started.
+    // TODO(fmassot): try to reduce the duration or use a wait until condition.
+    tokio::time::sleep(Duration::from_secs(3)).await;
```

Comment:
have you tries to use the mocking time stuff?

Reply:
not yet. Will try it.